io.github.nest-desktop.yml

```diff
@@ -0,0 +1,88 @@
+id: io.github.nest-desktop
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: "23.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node20
+  - org.freedesktop.Sdk.Extension.typescript
+command: nest-desktop-launch
+separate-locales: false
+rename-desktop-file: nest-desktop.desktop
+rename-icon: nest-desktop
+rename-appdata: nest-desktop
+finish-args:
+  # GPU acceleration if needed
+  - --device=dri
+  # Needs to save files locally
+  - --filesystem=home
+  # XShm access
+  - --share=ipc
+  # Needs to talk to the network:
+  - --share=network
+  # Display access (X11 or Wayland)
+  # - --socket=fallback-x11
+  - --socket=system-bus
```

Comment:
this is not allowed. you have to figure out the exact permissions needed.

Reply:
https://docs.flatpak.org/en/latest/sandbox-permissions.html#d-bus-access

Reply:
I removed `--socket=system-bus`

Reply:
that warning is harmless, but you most likely need to figure out what permissions are needed with --log-session-bus/ --log-system-bus

Reply:
```
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C2: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C3: -> org.freedesktop.DBus fake wildcarded AddMatch for org.nest_simulator.nest-desktop
C4: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.org.nest_simulator.nest-desktop
C5: -> org.freedesktop.DBus fake ListNames
B1: <- org.freedesktop.DBus return from C1
B2: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B3: <- org.freedesktop.DBus return from C2
*SKIPPED*
B4: <- org.freedesktop.DBus return from C3
*SKIPPED*
B5: <- org.freedesktop.DBus return from C4
*SKIPPED*
B6: <- org.freedesktop.DBus return from C5
C6: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C7: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C8: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C9: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Flatpak
C10: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
C11: -> org.freedesktop.portal.Flatpak call org.freedesktop.DBus.Properties.Get at /org/freedesktop/portal/Flatpak
B7: <- org.freedesktop.DBus return from C6
*SKIPPED*
B8: <- org.freedesktop.DBus return from C7
*SKIPPED*
B9: <- org.freedesktop.DBus return from C8
*SKIPPED*
B10: <- org.freedesktop.DBus return from C9
*SKIPPED*
B11: <- org.freedesktop.DBus return from C10
*SKIPPED*
B101: <- :1.4636 return from C11
C12: -> org.freedesktop.portal.Flatpak call org.freedesktop.DBus.Properties.Get at /org/freedesktop/portal/Flatpak
B102: <- :1.4636 return from C12
C1: -> org.freedesktop.DBus call org.freedesktop.DBus.Hello at /org/freedesktop/DBus
C2: -> org.freedesktop.DBus fake wildcarded AddMatch for org.freedesktop.portal
C3: -> org.freedesktop.DBus fake wildcarded AddMatch for org.nest_simulator.nest-desktop
C4: -> org.freedesktop.DBus fake wildcarded AddMatch for org.mpris.MediaPlayer2.org.nest_simulator.nest-desktop
C5: -> org.freedesktop.DBus fake ListNames
B1: <- org.freedesktop.DBus return from C1
B2: <- org.freedesktop.DBus signal org.freedesktop.DBus.NameAcquired at /org/freedesktop/DBus
B3: <- org.freedesktop.DBus return from C2
*SKIPPED*
B4: <- org.freedesktop.DBus return from C3
*SKIPPED*
B5: <- org.freedesktop.DBus return from C4
*SKIPPED*
B6: <- org.freedesktop.DBus return from C5
C6: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.DBus
C7: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Desktop
C8: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Documents
C9: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.Flatpak
C10: -> org.freedesktop.DBus fake GetNameOwner for org.freedesktop.portal.IBus
*SKIPPED*
B7: <- org.freedesktop.DBus return from C6
*SKIPPED*
B8: <- org.freedesktop.DBus return from C7
*SKIPPED*
B9: <- org.freedesktop.DBus return from C8
*SKIPPED*
B10: <- org.freedesktop.DBus return from C9
*SKIPPED*
B11: <- org.freedesktop.DBus return from C10
*SKIPPED*
C11: -> org.freedesktop.DBus call org.freedesktop.DBus.AddMatch at /org/freedesktop/DBus
B12: <- org.freedesktop.DBus return from C11
C12: -> org.freedesktop.portal.Flatpak call org.freedesktop.portal.Flatpak.Spawn at /org/freedesktop/portal/Flatpak
B104: <- :1.4636 return from C12
B105: <- :1.4636 signal org.freedesktop.portal.Flatpak.SpawnStarted at /org/freedesktop/portal/Flatpak
[7:1105/152709.732011:ERROR:bus.cc(407)] Failed to connect to the bus: Failed to connect to socket /run/dbus/system_bus_socket: No such file or directory
```

Reply:
Clicking on desktop file (on icon) works but not in terminal `flatpak run ..`.

Reply:
to see the system bus you need to also add `--system-talk=something` (not sure if that value will pass, but any valid dbus name is enough) along the `--log-system-bus` otherwise it's closed

Reply:
there is no such name on the system bus, so no. The only reason this make the message disappear is that it exposes a filtered system bus. But if you don't know what name is needed, put none. It's just a bug.

Two wrong don't make a right, but you don't even copy the wrong properly at it says `'--talk-name=org.freedesktop.Notifications'` (and it's not necessary either)